.github/workflows/vet_and_format.yml

```diff
@@ -0,0 +1,41 @@
+name: Vet and Format
+
+on: [push]
+      
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+jobs:
+  run-vet-gofmt:
+    name: Vet and format
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Verify dependencies
+        run: go mod verify
+
+      # ... acts as a wildcard: https://stackoverflow.com/a/13321739/10281310
+      - name: Build
+        run: go build -v ./...
+
+      - name: Run go vet
+        run: go vet ./...
+
+      - name: Run gofmt
+        uses: wearerequired/lint-action@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          # Enable linters
+          gofmt: true
+          gofmt_dir: ./...
```

Comment:
> Seems like the yml is incorrect somewhere, since it failed here. Have you seen this error before in the frontend repo @daltonfury42? I assumed the GitHub secret token doesn't have to be explicitly set up, could that be the reason?

What does `./...` mean? 

Lint-action is a github action written in js ([ref](https://github.com/wearerequired/lint-action/tree/master/src/linters) random folder).

The error we are getting `Cannot read property 'trim' of null` seems to be some `.trim()` call inside somewhere which is happening on a variable that happened to be null for some reason. That's what I think is happening.

Can we give a regular directory name to test if it's this line causing something? It's a very wild guess.

Remotely related: https://github.com/wearerequired/lint-action/issues/142

Reply:
... acts as a wildcard to get all files: (https://stackoverflow.com/a/13321739/10281310). I'll check with a single directory.

Reply:
Seems like this was the issue, it works with just the base dir.

Reply:
yeah, I guess the error was due to ... returning a list of folders or files, which the github action doesn't handle properly.